crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -242,16 +249,60 @@ mod tests {
             #[qproperty(T, name, cxx_name = "myName", rust_name = "my_name")]
             struct MyStruct;
         };
-        let property = mock_property(input);
 
-        assert_eq!(property.name.cxx_unqualified(), "myName");
-        assert_eq!(property.name.rust_unqualified(), "my_name");
+        // Should follow rust and cxx name
+        {
+            let property = mock_property(input.clone());
+            assert_eq!(property.name.cxx_unqualified(), "myName");
+            assert_eq!(property.name.rust_unqualified(), "my_name");
+        }
+
+        // Should ignore any auto casing
+        {
+            let auto_case = CaseConversion {
+                cxx: Some(Case::Snake),
+                rust: Some(Case::Camel),
+            };
+            let property = mock_property_case(input, auto_case);
+            assert_eq!(property.name.cxx_unqualified(), "myName");
+            assert_eq!(property.name.rust_unqualified(), "my_name");
+        }
+    }
+
+    #[test]
+    fn test_parse_named_property_casing() {
+        let input: ItemStruct = parse_quote! {
+            #[qproperty(T, my_name)]
+            struct MyStruct;
+        };
+
+        // cxx_name to camel
+        {
+            let auto_case = CaseConversion {
+                cxx: Some(Case::Camel),
+                rust: None,
+            };
+            let property = mock_property_case(input.clone(), auto_case);
+            assert_eq!(property.name.cxx_unqualified(), "myName");
+            assert_eq!(property.name.rust_unqualified(), "my_name");
+        }
+
+        // rust_name to camel
+        {
+            let auto_case = CaseConversion {
+                cxx: None,
+                rust: Some(Case::Camel),
+            };
+            let property = mock_property_case(input, auto_case);
+            assert_eq!(property.name.cxx_unqualified(), "my_name");
+            assert_eq!(property.name.rust_unqualified(), "myName");
+        }
```

Comment:
Add a test case for both auto_cxx_name and auto_rust_name here?
Or do we currently not support this?

Reply:
I'll add a test for this